sbt-scalajs-reflection/src/main/scala/be/doeraene/sjsreflect/sbtplugin/ReflectionLinker.scala

```diff
@@ -162,6 +162,14 @@ final class ReflectionLinker(underlying: GenLinker,
       JSObjectConstr(items)
     }
 
+    def makeClassArray()(implicit pos: ir.Position): Tree = {
+      val shouldEnable = shouldEnableOperation(ReflectEnumerateClass)
+
+      val items = infos.withFilter(shouldEnable).map(info => ClassOf(ClassType(info.encodedName)))
```

Comment:
Consider following the style used in `makeClassesByName()`, i.e., using a for comprehension:
```scala
       val items = for {
         info <- infos
         if shouldEnable(info)
       } yield {
         ClassOf(ClassType(info.encodedName))
       }
```